src/dipdup/config.py

```diff
@@ -908,6 +920,24 @@ def address_filter(self) -> Set[str]:
         return addresses
 
 
+@dataclass
+class OperationUnfilteredIndexConfig(IndexConfig):
+    """Operation index config
+
+    :param kind: always `operation`
+    :param handlers: List of indexer handlers
+    :param first_level: Level to start indexing from
+    :param last_level: Level to stop indexing at (DipDup will terminate at this level)
+    """
+
+    kind: Literal["operation"]
```

Comment:
Should be `operation_unfiltered` right?

Reply:
I was actually wondering if I had to change that or not. I'll update it!